components/event_dispatcher/introduction.rst

```diff
@@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
 ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
 e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
 avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
-il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
+il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
 nell'oggetto distributore. Ecco degli esempi:
```

Comment:
La SERP (rigorosamente su scheda anonima) di Google per "caricamento pigro" restituisce pagine sul lazy loading in 8 dei primi 10 risultati.